config/locales/en.yml

```diff
@@ -991,11 +991,15 @@ en:
         email_form:
           label: Send an email
           submit: Send
+        email_opted_out: Client has opted out of emails.
         no_email_address: Client has not provided an email address.
+        no_email_and_opt_out: Client has not provided an email address and opted out of emails.
+        no_sms_and_opt_out: Client has not provided an SMS phone number and opted out of text messages.
         no_sms_number: Client has not provided an SMS phone number.
-        no_way_to_communicate: Client has not provided contact info for digital communication.
+        no_way_to_communicate: Client has not provided contact info for digital communication and/or has opted-out.
         preference_intro: "%{preferred_name} has opted in to contact by:"
         response_not_needed: Mark as "Response not needed"
+        sms_opted_out: Client has opted out of text messages.
```

Comment:
likewise with this one i feel like it's more that they didn't check the box that said that was how they want to be contacted. my additional nitpick with this copy is that there's also the twilio-level opt out which i don't think we actually capture and this could look like it covers that when it doesn't.